stl/inc/atomic

```diff
@@ -1158,6 +1158,24 @@ struct _Atomic_integral<_Ty, 8> : _Atomic_storage<_Ty> { // atomic integral oper
 #endif // _M_IX86
 };
 
+#if _HAS_CXX17
+#if 1 // TRANSITION, ABI
+
+template <class _Ty>
+static constexpr bool _Is_always_lock_free = sizeof(_Ty) <= 8 && (sizeof(_Ty) & sizeof(_Ty) - 1) == 0;

```

Comment:
Although `-` has higher precedence than `&`, we conventionally use parentheses in the immediate vicinity of bitwise operators because their precedence is famously confusing.

Additionally, this is at namespace scope, so this should be `inline constexpr` instead of `static constexpr`.

```suggestion
inline constexpr bool _Is_always_lock_free = sizeof(_Ty) <= 8 && (sizeof(_Ty) & (sizeof(_Ty) - 1)) == 0;
```